docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -117,6 +118,96 @@ In this case, the property value is a semicolon separated list containing the
 |    :end-before: <!--><-->                                                                                            |
 +----------------------------------------------------------------------------------------------------------------------+
 
+.. _property_policies_physical_data:
+
+Physical Data in Discovery Information
+""""""""""""""""""""""""""""""""""""""
+
+It is possible to include the information conveyed in the :ref:`statistics_topic_names_physical` into the participant
+discovery message, a.k.a *DATA[p]*.
+This is done by setting the following properties within the :ref:`propertypolicyqos`:
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - PropertyPolicyQos name
+     - PropertyPolicyQos value
+     - Default value without |br| ``FASTDDS_STATISTICS``
+     - Default value with |br| ``FASTDDS_STATISTICS``
+   * - ``"fastdds.physical_data.host"``
+     - Name of the host computer in which |br| the application runs
+     - Not set
+     - ``""``
+   * - ``"fastdds.physical_data.user"``
+     - Name of the user running the application
+     - Not set
+     - ``""``
+   * - ``"fastdds.physical_data.process"``
+     - Name of the process running the application
+     - Not set
+     - ``""``
+
+Whenever any of these properties is defined within the |DomainParticipantQos-api|, the |DomainParticipant-api| *DATA[p]*
+will contained the set value.
+Furthermore, if any of these properties is set to a value of ``""`` (the default when ``FASTDDS_STATISTICS`` is
```

Comment:
Maybe a link to the CMake options section might help the reader understand what you are referring to.